collectors/PropertiesCollector.py

```diff
@@ -0,0 +1,95 @@
+from BaseCollector import BaseCollector
+import logging
+
+logger = logging.getLogger('vrops-exporter')
+
+
+class PropertiesCollector(BaseCollector):
+
+    def __init__(self):
+        super().__init__()
+
+    def get_uuids(self):
+        raise NotImplementedError("Please Implement this method")
+
+    def set_labels(self, resource_id: str, project_ids: list):
+        raise NotImplementedError("Please Implement this method")
+
+    def collect(self):
+        logger.info(f'{self.name} starts with collecting the metrics')
+
+        token = self.get_target_tokens()
+        token = token.setdefault(self.target, '')
+        if not token:
+            logger.warning(f'skipping {self.target} in {self.name}, no token')
+            return
+
+        uuids = self.get_uuids()
```

Comment:
shouldn't we rather use `get_project_ids` naming wise?

Reply:
no, these are resource uuids, which are set to the actual resource uuids in the respective classes.